packages/flutter/ios/flutter_breez_liquid.podspec

```diff
@@ -25,16 +26,18 @@ Pod::Spec.new do |spec|
   spec.prepare_command = <<-CMD
     cd Frameworks
     rm -rf #{framework_name}
-
-    if [ ! -f #{local_zip_name} ]
-    then
-      wget #{url} -O #{local_zip_name} || curl -L #{url} -o #{local_zip_name}
-    fi
-
     unzip #{local_zip_name}
     cd -
   CMD
   
   spec.ios.deployment_target = '12.0'
   spec.osx.deployment_target = '10.11'
+
+  spec.dependency 'Flutter'
+  spec.platform = :ios, '11.0'
```

Comment:
This is not needed because the deployment_targets are set

https://guides.cocoapods.org/syntax/podspec.html#platform
```
  spec.ios.deployment_target = '12.0'
  spec.osx.deployment_target = '10.11'
```

Reply:
Removed f79d2122613116aacc7f9332520c8d1a95becc60